### PR TITLE
chore: updated analyze.go

### PR DESCRIPTION
**Pull request by chetanguptaa**
Closes # 

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

**Comment by matthisholleville** [comment]
Can you please Add a description of your changes ?

File: cmd/analyze/analyze.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ var (
 // AnalyzeCmd represents the problems command
 var AnalyzeCmd = &cobra.Command{
 	Use:     "analyze",
-	Aliases: []string{"analyse"},
+	Aliases: []string{"analyze"},
```
Comment (AlexsJones) [inline cmd/analyze/analyze.go:32]: The alias is on-purpose for EN/US versions of the word

**Review by AlexsJones — COMMENTED** [review]
Please see my comment, also it's not clear what the changes L86-L89 improve. Thanks!

**Comment by chetanguptaa** [comment]
Ok

**Pull request closed without merging**